Add feature to easily use input files in Sphinx' output location (#2)
melexis/treemap: doc/usage.rst

=====
Usage
=====

A treemap can be generated using the ``treemap`` directive:

.. code-block:: rest

    .. treemap:: parent_name parent_color
        :input: <<outdir>>/coverage_data.xml
        :limits: 90 80 70 0
        :colors: lightgreen orange yellow darkred

:parent_name:

    Mandatory name for the parent group.

:parent_color: *optional*

    Optional color for the parent group, lightgrey by default.

:input: *single argument*

>     Path to the input file. The literal string ``<<outdir>>`` will be replaced by Sphinx' output directory.

Reviewer: This is strange wording. Maybe betrer

```suggestion
    Path to the input file. The path is relative to Sphinx' output directory  ``<<outdir>>`` .
```

Author: That's not what I meant. I clarified it in 5aadd32738741af1a7cc811d7ba88a7172217c34.
